MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/service/assetSentiment/AssetSentimentDaoTest.java

```diff
@@ -0,0 +1,71 @@
+package com.google.moviestvsentiments.service.assetSentiment;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import android.content.Context;
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule;
+import androidx.room.Room;
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import com.google.moviestvsentiments.model.Asset;
+import com.google.moviestvsentiments.model.AssetType;
+import com.google.moviestvsentiments.service.database.SentimentsDatabase;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class AssetSentimentDaoTest {
+
+    @Rule
+    public InstantTaskExecutorRule instantTaskExecutorRule = new InstantTaskExecutorRule();
+
+    private AssetSentimentDao assetSentimentDao;
+    private SentimentsDatabase database;
+
+    @Before
+    public void createDatabase() {
+        Context context = ApplicationProvider.getApplicationContext();
+        database = Room.inMemoryDatabaseBuilder(context, SentimentsDatabase.class)
+                .allowMainThreadQueries().build();
+        assetSentimentDao = database.assetSentimentDao();
+    }
+
+    @After
+    public void closeDatabase() {
+        database.close();
+    }
+
+    @Test
+    public void addAndGetAsset_withIncorrectId_returnsNull() {
```

Comment:
nit: Please restructure test to three sections: setup, execute, verify. (https://drive.google.com/file/d/1C-0r_j-SIJf4_j-_cY9V8R-1h6mfuUWd/view?usp=sharing)